### PR TITLE
[MM-37616] Self-managed: Update email template to new branding - Welcome and Verification email

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
This PR updates the welcome and verify email templates to new branding

Screenshot 2021-08-09 at 10 15 46

-----

Screenshot 2021-08-09 at 10 14 10

#### Ticket Link
[MM-37616](https://mattermost.atlassian.net/browse/MM-37616?atlOrigin=eyJpIjoiZGM4Mjc4M2U0NGU1NDdlYWJmOWU1YzJmZTMzZTViNzYiLCJwIjoiaiJ9)

#### Release Note

```release-note
NONE

```

### CODE REVIEW

File: templates/verify_body.html [hunk 1]
```diff
@@ -475,7 +475,8 @@
                               <tr>
                                 <td align="center" class="footerInfo" style="font-size:0px;padding:0px;word-break:break-word;">
                                   <div style="font-family:Open Sans, sans-serif;font-size:13px;line-height:1;text-align:center;color:#000000;">{{.Props.QuestionInfo}}
-                                    <a href="mailto:{{.Props.SupportEmail}}"> {{.Props.SupportEmail}}</a>
+                                    <a href="mailto:{{.Props.SupportEmail}}">
+                                      {{.Props.SupportEmail}}</a>
```
Comment (pablovelezvidal) [inline templates/verify_body.html:479]: maybe we want to put the closing anchor tag in a new line?

**Review by pablovelezvidal — APPROVED** [review]
LGTM! 🚢

**Review by nickmisasi — APPROVED** [review]

**Review by neallred — APPROVED** [review]

**Review by matthewbirtch — COMMENTED** [review]
I'm not sure which PR to comment on here, so I will comment on both. 

I can only review based on the screenshots, but it looks like some changes still need to be updated:
- The headline font is now Metropolis in the emails
- The button/link color is now #1C58D9
- The logo color has been updated to #3F4350 as well
- Text color is now #3F4350

Also, is there a way test emails could be sent to me for closer UI review?

File: templates/verify_body.html (hunk 1 above)
Comment (AGMETEOR) [reply templates/verify_body.html:479]: This file is auto-generated

Comment (pablovelezvidal) [reply templates/verify_body.html:483]: gotcha

**Comment by matthewbirtch** [comment]
Approving this for now since the logo change and color changes were made based on the code changes I can see. I don't want to be a bottleneck, but we'll need to figure out how we can review these moving forward

**Pull request merged**